chronoline/chronoline.js

```diff
@@ -394,13 +398,47 @@ function Chronoline(domElement, events, options) {
         // DRAWING
         t.floatingSet = t.paper.set();
         t.sectionLabelSet = t.paper.set();
+
+        //attach background click events function
+        t.myCanvas.onclick = function (e) {
+            //if not event or section click.
+            if (e.target.nodeName != 'circle' && e.target.nodeName != 'rect') {
+                var clickDate = new Date(t.pxToMs(e.clientX));
+                clickDate.stripTime();
+                t.backgroundClick(clickDate);
```

Comment:
I thought about doing passing the event. But it would have been inconsistent. Section and event clicks would have Raphael event object, background click would have a regular dom event. The Raphael event has the data attached to it so I would be essentially passing the data twice, or remove the data parameter as well. I didn't know how to cast one type of event into the other type and I felt that if I passed them both to different event handlers I would have to provide documentation, which I was to lazy to write. So I just didn't pass the event.

I am probably just over thinking this. It's no problem, I can add the event to the event in if you like.